test/util/bspNet/api.ts

```diff
@@ -3,7 +3,7 @@ import type { BspNetApi } from "./types";
 import type { SubmittableExtrinsic } from "@polkadot/api/types";
 import type { ISubmittableResult } from "@polkadot/types/types";
 import type { KeyringPair } from "@polkadot/keyring/types";
-import { createBucket, sealBlock, sendLoadFileRpc } from "./helpers";
+import { createBucket, getForestRoot, sealBlock, sendLoadFileRpc } from "./helpers";
 import { assertEventPresent } from "../asserts";
 import type { EventRecord, H256 } from "@polkadot/types/interfaces";
 
```

Comment:
Given we are adding more RPC methods and helpers, might be worth starting to add these directly to the API object. 

https://polkadot.js.org/docs/api/start/rpc.custom

We can add the custom RPC interfaces here and we can get the proper types throughout the code. @TDemeco has experience here as this is what they did extensively at a previous firm